utils/near-rate-limiter/src/framed_read.rs

```diff
@@ -368,4 +399,26 @@ mod tests {
 
         assert_eq!(semaphore.available_permits(), 1);
     }
+
+    #[test]
```

Comment:
do we have any test for FrameReader? if not - can you add a very basic one, and see that throttler results are what we expect?

Reply:
@mm-near I created a new PR for adding test: https://github.com/near/nearcore/pull/5758
It requires more work, I need to figure out why the test fails.